### ISSUE TITLE
performace of triplet loss

### ISSUE THREAD

**Issue by log26** [bnulihaixia/Deep_metric#9]
Hello! Have you ever reproduced the results for triplet loss or N-pair loss? I found that the performace of ImageNet pretrained model is  even better than the model further trained with triplet loss on CUB or CAR.

**Comment by bnulihaixia** [comment]
The model here is incetption-BN and dimension is 512 while in the paper of triplet loss or N-pair loss, the dimension is 64, and the model maybe inception without BN. 

In my version of triplet loss, the result is about 62+ rank-1 for CUB under 512 dimension, I have not recorded the exact result.
